subsys/bluetooth/host/att.c

```diff
@@ -1942,6 +1942,28 @@ static att_type_t att_op_get_type(u8_t op)
 		return ATT_COMMAND;
 	}
 
+#if !defined(CONFIG_BT_GATT_CLIENT)
```

Comment:
Complete the list and just get rid of the rest of the function?

Reply:
Yep, Ive been thinking about that, though that means one has to remember to add new opcodes there as well but I guess it is faster to use the switch statement as well.

Reply:
I think the switch statement is better, it's more robust to ifdefs in the handlers array.